### PR TITLE
Modifiable values consistent across drivers

### PR DESCRIPTION
**Pull request by rishabhatdell**
# Description

- This PR made modifiable values in values.yaml consistent across drivers
- Fixed CVE-2024-45338

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1647|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility
- [ ] I have executed the relevant end-to-end test scenarios

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Installed Powerflex driver with new yaml
- [x] Built Image
- [x] Ran e2e for a driver
image

### CODE REVIEW

**Review by shanmydell — DISMISSED** [review]

**Review by mgandharva — COMMENTED** [review]
@rishabhatdell Doen't it require the changes in E2E sample files?

**Review by suryagupta4 — APPROVED** [review]

File: samples/storage_csm_powerflex_v2110.yaml [hunk 1]
```diff
@@ -47,8 +47,8 @@ spec:
       # 'k8s' represents a string prepended to each volume created by the CSI driver
```
Comment (adarsh-dell) [inline samples/storage_csm_powerflex_v2110.yaml:47]: @rishabhatdell This comment also needs to get updated, what do you think?

File: samples/storage_csm_powerflex_v2120.yaml [hunk 2]
```diff
@@ -54,8 +54,8 @@ spec:
       # 'k8s' represents a string prepended to each volume created by the CSI driver
```
Comment (adarsh-dell) [inline samples/storage_csm_powerflex_v2120.yaml:54]: Same comment as above, and also for the below files.

File: operatorconfig/driverconfig/powerflex/v2.11.0/controller.yaml [hunk 3]
```diff
@@ -143,7 +143,7 @@ spec:
           args:
             - "--csi-address=$(ADDRESS)"
             - "--feature-gates=Topology=true"
-            - "--volume-name-prefix=k8s"
+            - "--volume-name-prefix=csivol"
```
Comment (donatwork) [inline operatorconfig/driverconfig/powerflex/v2.11.0/controller.yaml:146]: Do you really need the "vol" in the prefix? It is already a volume and the management UIs will not mix the display of volumes with other objects, The "vol" part seems redundant.

Comment (rishabhatdell) [reply operatorconfig/driverconfig/powerflex/v2.11.0/controller.yaml:146]: All drivers use this as the default prefix, which has already been approved and merged in Helm charts.

**Review by suryagupta4 — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]

**Review by gilltaran — APPROVED** [review]
LGTM

**Pull request merged**